tfe/data_source_variables.go

```diff
@@ -0,0 +1,124 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFEWorkspaceVariables() *schema.Resource {
+	varSchema := map[string]*schema.Schema{
+		"category": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"hcl": {
+			Type:     schema.TypeBool,
+			Computed: true,
+		},
+		"id": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"name": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"sensitive": {
+			Type:     schema.TypeBool,
+			Computed: true,
+		},
+		"value": {
+			Type:      schema.TypeString,
+			Computed:  true,
+			Sensitive: true,
+		},
+	}
+	return &schema.Resource{
+		Read: dataSourceVariableRead,
+
+		Schema: map[string]*schema.Schema{
+			"environment": {
```

Comment:
As I was using the resource, I found myself expecting this to mirror the category and be named "env". Could you rename this schema field (as well as in the docs?)

Reply:
Did the changes and the tests output went well but please review them because I'm working from my moms setup and i'm not comfortable on it, sorry about that and thanks, @brandonc

Reply:
Todayt I was able to complete all my tests it looks fine to me
Regards